### ISSUE TITLE
Some validation output of promoted enum values uses the _KHR versions instead of the core versions

### ISSUE THREAD

**Issue by mark-lunarg** [KhronosGroup/Vulkan-ValidationLayers#2398]
Noted in `stateless_validation.h,` in situations like the one below.  Aliases should be changed to the core versions where it is not specifically related to a KHR-specific version.

```
 if (!separate_depth_stencil_layouts) {
                if (pCreateInfo->pAttachments[i].initialLayout == VK_IMAGE_LAYOUT_DEPTH_ATTACHMENT_OPTIMAL_KHR ||
                    pCreateInfo->pAttachments[i].initialLayout == VK_IMAGE_LAYOUT_DEPTH_READ_ONLY_OPTIMAL_KHR ||
                    pCreateInfo->pAttachments[i].initialLayout == VK_IMAGE_LAYOUT_STENCIL_ATTACHMENT_OPTIMAL_KHR ||
                    pCreateInfo->pAttachments[i].initialLayout == VK_IMAGE_LAYOUT_STENCIL_READ_ONLY_OPTIMAL_KHR) {
                    vuid = use_rp2 ? "VUID-VkAttachmentDescription2-separateDepthStencilLayouts-03298"
                                   : "VUID-VkAttachmentDescription-separateDepthStencilLayouts-03284";
                    skip |= LogError(
                        device, vuid,
                        "%s: pCreateInfo->pAttachments[%d].initialLayout must not be VK_IMAGE_LAYOUT_DEPTH_ATTACHMENT_OPTIMAL_KHR, "
                        "VK_IMAGE_LAYOUT_DEPTH_READ_ONLY_OPTIMAL_KHR, VK_IMAGE_LAYOUT_STENCIL_ATTACHMENT_OPTIMAL_KHR, or "
                        "VK_IMAGE_LAYOUT_STENCIL_READ_ONLY_OPTIMAL_KHR",
                        func_name, i);
}
```

VUID for this example is 03298/03284, but the entire file should be scrubbed.

**Comment by sfricke-samsung** [comment]
[VK_VERSION_1_1.txt](https://github.com/KhronosGroup/Vulkan-ValidationLayers/files/5700267/VK_VERSION_1_1.txt)
[VK_VERSION_1_2.txt](https://github.com/KhronosGroup/Vulkan-ValidationLayers/files/5700268/VK_VERSION_1_2.txt)

These are the generated files when you build the Vulkan spec that references the appendix with what is different/promoted... might be easier to run a regex...which I know I have laying around because I have done this before... think I just volunteered myself :)

**Comment by sfricke-samsung** [comment]
so found script and ran it (on non-generated files), there are like 1200 places in ./tests and 600 places in ./layers that use KHR/EXT alias instead of the new 1.1 or 1.2 core version... I can quick get a PR up on just `stateless_validation.h` and you can decide if you want it else where

**Comment by mark-lunarg** [comment]
Fixed by #2402 and #2409.  Thanks guys!
